bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -2108,9 +2136,10 @@ private static ImageData applyMask(ImageData source, ImageData mask) {
 	return imageData;
 }
 
-void init(ImageData i) {
+
+void init(ImageData i, Integer zoomLevel) {
```

Comment:
As we call it zoom or nativeZoom anywhere else, we should call it zoom here as well